sql/2021/javascript/ajax_request_per_page.sql

```diff
@@ -0,0 +1,22 @@
+SELECT
+  client,
+  percentile,
+  APPROX_QUANTILES(ajax_requests_total, 1000)[
+    OFFSET(percentile * 10)] AS ajax_requests_total
+FROM (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    CAST(JSON_EXTRACT(JSON_EXTRACT_SCALAR(payload,
+          '$._javascript'),
+        '$.ajax_requests.total') AS INT64) AS ajax_requests_total
+  FROM
+    `httparchive.sample_data.pages_*` ),
```

Comment:
Throughout the PR, make sure you're querying the full dataset.

@NishuGoel have you been added to the HA project billing account yet? (cc @paulcalvano)

```suggestion
    `httparchive.pages.2021_07_01_*` ),
```

Reply:
@paulcalvano @rviscomi 
I don't seem to have access to HA project billing account. Querying only sample_data currently.

Reply:
I've just added your gmail account to the project. Access BigQuery via this link with "project=httparchive" set (and confirm that you see "HTTP Archive" listed as the current project in the page header, that's the project that will be billed so you don't want it to be a personal project!):

https://console.cloud.google.com/bigquery?project=httparchive

![image](https://user-images.githubusercontent.com/1120896/136101423-630f7334-9717-4f49-87af-10fded4b200a.png)


You should now be able to query the full dataset without incurring any costs personally.